src/main/java/com/ss/training/utopia/agent/service/AgentService.java

```diff
@@ -26,4 +26,81 @@
         return bookings;
     }
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    public Booking cancelBooking(Booking booking) {
+        // HOW TO DO A STRIPE REFUND
+
+        // STORE A STRIPE KEY
+        // Stripe.apiKey = "sk_test_4eC39HqLyjWDarjtT1zdp7dc";
+
+        // GET THE PRICE FROM FLIGHTS TABLE
+        // float price = flightDAO.getByFlightId().getPrice();
+
+        // CREATE A REFUND OBJECT
+        // Refund refund = Refund.create(RefundCreateParams.builder()
+        //   .setPaymentIntent("pi_Aabcxyz01aDfoo")
+        //   .build());
+
+        // Handle the Stripe request
+
+        booking.setActive(false);
+
+        bookingDAO.save(booking);
```

Comment:
I think there should be some exception handling here in case the database operation fails.